crates/block-producer/src/runner.rs

```diff
@@ -136,6 +139,10 @@ pub fn run(config: Config) -> Result<()> {
             eth_lock_script_type_hash.unpack(),
             Box::new(Secp256k1Eth::default()),
         );
+        account_lock_manage.register_lock_algorithm(
+            eth_lock_script_type_hash.unpack(),
```

Comment:
We do not enable this lock in the current release, Pls remove this

Reply:
Done